website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -72,18 +72,57 @@ Provide interpreters upon request and in advance. We also recommend having two s
 
-Visibility
-Ensure adequate lighting for interpreters. In particular, keynote rooms tend to be darker, with bright spotlights on the stage. Consider adding frontal or overhead lighting in the interpreter seating area. Here's an anecdote from one attendee:
+#### Include interpreters in recording
 
-> "At one point during the keynote, one interpreter sat directly below another interpreter, shining a phone flashlight from below to improve visibility." 
+If you record your sessions, be sure the interpreters are on screen so deaf attendees can benefit from your accessibility effort even when watching them on-demand.
+
+### Captions
+
+There is no need to provide interpreters for every session, but captions should be available for all sessions. While there are lots of live captions software options, not all work well. Using a tool with low accuracy can be very confusing for deaf/hoh individuals, so select one that has proven to work.
+
+Some caption tools work for virtual events only, others for in-person and virtual. 
+
+#### Some considerations
+
+Captions at the bottom of the screen are not visible unless attendees are sitting at the front. Consider adding them at the top. If that isn't feasible,  you might want to place the screen higher to ensure people in the back can also read them. 
+
+Some captions are highly **accurate but lag** significantly. That creates a disconnect between the message received and the speaker/audience (not syncing with body language, laughter, etc.), which can create a feeling of ‘missing out’
+
+Unlike hearing people, deaf/hoh individuals can only focus on the captions or look at the speaker/slides (e.g., a technical graphic). So if they do look away, they will miss what's said. Hence, leaving content on the screen longer by **adding more than one line** will help people rely on captions. 
+
+Every session should have captions in the main conference language **on the screen by default** (vs. captions available on the phone). Reading captions on-screen makes attendees feel more engaged than looking at their phones (speakers might think they aren't interested). With a phone, attendees are constantly required to shift focus between their phones and speaker(s) and will miss parts of the message. This is even more critical when there are live demos because, to understand what is going on, attendees must see what's on-screen and read captions at the same time. Captions on the phone make that impossible. 
+
+#### Caption use cases beyond deaf/hoh
+
+Some attendees can hear but may be on the autism spectrum and have a hard time processing auditory cues — captioning is incredibly beneficial for them, too. Additionally, non-native speakers who aren’t fluent can benefit from having words on the screen.
```

Comment:
Some attendees can hear but may be neurodivergent ( e.g. on the autism spectrum, with ADHD, and more) and have a hard time processing auditory cues — captioning is incredibly beneficial for them, too. Additionally, non-native speakers who aren’t fluent can benefit from having words on the screen.